students/charlie_england/lesson08/assignment/tests/test_inventory.py

```diff
@@ -2,18 +2,41 @@
     Autograde Lesson 8 assignment
 
 """
-
+import sys
+sys.path.insert(1,"..//src")
```

Comment:
You should try to avoid using sys.path.insert to fix import issues. You should try to write things such that they are in relation to the assignment directory and run your actual python file from the assignment directory.